src/gmastertrack.cxx

```diff
@@ -169,8 +169,15 @@ static void gmastertrack_button_callback(Fl_Widget *w, void *data)
 
 
 	} else if ( strcmp( w->label(), "Tap" ) == 0 ) {
-		EventTimeTempoTap e;
-		writeToDspRingbuffer( &e );
+		if ( Fl::event_button() ==FL_RIGHT_MOUSE) {
+			const char* answer = fl_input("Enter BPM value: ");
```

Comment:
I belive this will crash in the exact same way you reported on the other thread? Try clicking cancel and see what happens...

Reply:
Fixed and pushed, thanks